### DISCUSSION TITLE
Is the lack of password to access webcam feature on port 81 wanted ?

### DISCUSSION THREAD

**Discussion by kaetir** [arendst/Tasmota#19935] [Q&A] [answered]
I found that even with a WebAdmin password set, it is still possible to access http://ip_tasmota:81/stream (with a compatible and configured module)

Shouldn't it be protected by the same password protection ? 

I can see this as a potential security vulnerability for surveillance camera applications.

**Comment by sfromis** [comment, marked as answer]
Probably just something no one worried about, as Tasmota does not try to provide "real security" on a network, only very basic "child proofing" sort of a guardrail against admin stuff, but not very secure or having robust privilege system.

**Reply by kaetir** [reply]
Thanks for the information. 

It's a little sad to not have better "real security" for such a wonderful project. 

I know that most board from the ESP family are not made for security application, but they provide hobbyists a very affordable way to tinker.

**Reply by sfromis** [reply]
In case of successful intrusion into my network, the Tasmota devices would be very low on my list of things to be afraid of. OTOH, not sharing the network with users having more "risky habits" makes things simpler.
